package-lock.json

```diff
@@ -87,7 +87,7 @@
         "debug": "^4.1.0",
         "gensync": "^1.0.0-beta.1",
         "json5": "^2.1.2",
-        "lodash": "^4.17.20",
+        "lodash": "^4.17.19",
```

Comment:
Uurgh this again!!

Reply:
Ok now I am not actually sure why this is happening :(

Reply:
This looks to have been fixed now